packages/eslint-tool/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# See https://help.github.com/articles/about-codeowners/ for more information about this file.
+
+- @financial-times/reliability
```

Comment:
Don't know if this file is necessary? or if I've used the right name path for our team?